style.css

```diff
@@ -0,0 +1,496 @@
+.header-background {
+  display: flex;
+  position: absolute;
+  width: 475px;
+  height: 916px;
+  left: 0;
+  top: 1px;
+
+  /* Neutrals/N0 */
+
+  background-color: #fff;
+}
+
+.dkl {
+  float: right;
+  margin-right: 90px;
+}
+
+.sec3rlink23 {
+  list-style-type: none;
+}
+
+.dkk {
+  position: absolute;
+  width: 48px;
+  height: 48px;
+  left: 12px;
+}
+
+.dd {
+  display: inline;
+}
+
```

Comment:
- [Optional] Kindly note that it is best practice to use meaningful class names for your elements.

- [Optional] Please add a font family from the design template. In the design template, '''Poppins' was used as the font family. You can get it from google fonts.